dockerv2.go

```diff
@@ -0,0 +1,49 @@
+package txtdirect
+
+import (
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+)
+
+var dockerRegexs = map[string]string{
+	"v2":        "^\\/?v2\\/?$",
+	"_catalog":  "^/v2/_catalog$",
+	"tags":      "^/v2/(.*)/tags/(.*)",
+	"manifests": "^/v2/(.*)/manifests/(.*)",
+	"blobs":     "^/v2/(.*)/blobs/(.*)",
+}
+
+var DockerRegex = regexp.MustCompile("^\\/v2\\/(.*\\/(tags|manifests|blobs)\\/.*|_catalog$)")
+
+func redirectDockerv2(w http.ResponseWriter, r *http.Request, rec record) error {
+	path := r.URL.Path
+	v2Regex, err := regexp.Compile(dockerRegexs["v2"])
+	if err != nil {
+		panic(err)
+	}
+	if v2Regex.MatchString(path) {
+		_, err := w.Write([]byte(http.StatusText(http.StatusOK)))
```

Comment:
Header is missing.
```
		w.Header().Set("Docker-Distribution-Api-Version", "registry/2.0")
```

Reply:
*above the status being written

Reply:
Header is still missing.

Reply:
Probably got removed by hitting ctrl+z too many times :laughing: